registry/storage/driver/gcs/gcs.go

```diff
@@ -1,18 +1,17 @@
+//go:build include_gcs
+// +build include_gcs
+
 // Package gcs provides a storagedriver.StorageDriver implementation to
 // store blobs in Google cloud storage.
 //
 // This package leverages the google.golang.org/cloud/storage client library
-//for interfacing with gcs.
+// for interfacing with gcs.
 //
 // Because gcs is a key, value store the Stat call does not support last modification
 // time for directories (directories are an abstraction for key, value stores)
 //
 // Note that the contents of incomplete uploads are not accessible even though
 // Stat returns their length
-//
-//go:build include_gcs
-// +build include_gcs
-
 package gcs
```

Comment:
Looks like we didn't fix these on the `main` branch (still has the empty line after, and the build-tags in the wrong spot); https://github.com/distribution/distribution/blob/983358f8e2509bf8ae196a8e135180a2c5b11264/registry/storage/driver/gcs/gcs.go#L12-L16

Reply:
Done in https://github.com/distribution/distribution/pull/3950